test/features/step_definitions/sdk_steps.rb

```diff
@@ -41,22 +46,43 @@
 
 Then(/^I should reach the complete step$/) do
   steps %Q{
-    Then page_title () should contain "Verification complete"
-    Then I should not see "back"
+    Then page_title () should contain "#{i18n.translate('complete.message')}"
```

Comment:
I think you can remove all `should contain` in favour of the new `should include`